hyperjumptech/monika
# Feat: 541 change create config to yaml

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Need to change create config from har/postman config result into yaml

## How did you implement / how did you fix it  
1. change the translated config result into yaml
2. set created file into yml file

## How to test  
1. set a har file / postman collection file
2. run the create config from har file command `npm start -- --create-config --har 'path/to/har_file'`
3. make sure there is created `monika.yml` file
4. try the config by running the application using created config `npm start -- -c monika.yml`
